src/D2L.CodeStyle.Analyzers/ApiUsage/ServiceLocator/OldAndBrokenServiceLocatorAnalyzer.cs

```diff
@@ -133,26 +132,26 @@ TypeWhitelist typeWhitelist
 
 				SemanticModel model = context.Compilation.GetSemanticModel( typeSyntax.SyntaxTree );
 
-				bool usesDisallowedTypes = typeSyntax
+				bool usesDisallowededTypes = typeSyntax
 					.DescendantNodes()
 					.OfType<IdentifierNameSyntax>()
-					.Any( syntax => IdentifierIsOfDisallowedType( model, disallowedTypes, syntax ) );
+					.Any( syntax => IdentifierIsOfDisallowededType( model, disallowededTypes, syntax ) );
 
-				if( usesDisallowedTypes ) {
+				if( usesDisallowededTypes ) {
 					return;
 				}
 			}
 
 			if( diagnosticLocation != null ) {
-				typeWhitelist.ReportEntryAsUnnecesary(
+				typeAllowedList.ReportEntryAsUnnecesary(
 					entry: namedType,
 					location: diagnosticLocation,
 					report: context.ReportDiagnostic
 				);
 			}
 		}
 
-		private static bool IdentifierIsOfDisallowedType(
+		private static bool IdentifierIsOfDisallowededType(

```

Comment:
?

Reply:
ctrl+F disalloweded in this PR

Reply:
Bugger. Yep, At one point I did a find-and-replace to correct an error I made, but I didn't account for cases where we were already using "allowed". I'll fix these.